app/controllers/idv/in_person/address_controller.rb

```diff
@@ -0,0 +1,67 @@
+module Idv
+  module InPerson
+    class AddressController < ApplicationController
+      include IdvStepConcern
+
+      before_action :render_404_if_not_in_person_residential_address_controller_enabled
```

Comment:
@soniaconnolly wanted to poke you about some of the `before_action`s we use in other controllers as defined in `idv_step_concern.rb` https://github.com/18F/identity-idp/blob/main/app/controllers/concerns/idv_step_concern.rb#L90-L109
Are these needed to make sure users who update the url manually are taken to the correct page given what steps they've completed? Do we need to include any of these before actions in this controller?

Reply:
Confusingly, `confirm_address_step_complete` refers to the phone step.

You'll want an `in_person_address_step_needed` and a check for whether the prior in_person step is complete.

Reply:
ok thank you for that info!